ext/devcontainer/src/azd-persistence/README.md

```diff
@@ -0,0 +1,17 @@
+# Azure Developer CLI - Persistence
+
+Preserve ~/.azd folder across container instances.
+
+## Example Usage
+
+```json
+"features": {
+    "ghcr.io/azure/azure-dev/azd-persistence:0": {}
```

Comment:
Does this need to be a sep feature?  I was thinking it would just be built in

Reply:
I don't think that it should be part of the installation. As a regular user, I expect that my container is back to its original state whenever I re-build the devcontainer. That's why the user should `opt-in ` to features like this, where they explicitly want to change the regular behavior.